GettingStarted.md

```diff
@@ -0,0 +1,162 @@
+# Introduction
+Welcome to impress.js! This presentation framework allows you to create stunning presentations with the power of CSS3 transformations. It is inspired by the idea behind [prezi.com](https://prezi.com). 
+**NOTE:** This Guide is not made for you, if you have never written HTML and/or CSS before. Knowing your way around in JavaScript certainly helps, but is not a necessity. You may still continue this tutorial and try to understand what we do as you go. 
+
+Also, if you are an experienced (or sorta experienced user, anyway), you might prefer the [DOCUMENTATION](DOCUMENTATION.md).
+
+# Getting started with impress.js
+## Installation / acquiring the framework
+You first need to decide if you want to use the bleeding edge (most up to date, generally very reliable as well, but may contain more problems) or the stable release. You can use one of the methods mentioned below or the direct download link to only the javascript file. The benefit is, you don't have to find the file in the repository and you can copy the link and directly load the file from the cdn. 
+
```

Comment:
For a getting started guide, keep it simple and avoid asking the user to decide. YOU should decide. Just show one option. Which one would you prefer?